types/node/http.d.ts

```diff
@@ -101,11 +101,13 @@ declare module "http" {
 
         setTimeout(msecs?: number, callback?: () => void): this;
         setTimeout(callback: () => void): this;
-        maxHeadersCount: number;
+        // https://nodejs.org/api/http.html#http_server_maxheaderscount
```

Comment:
Uh, please use actual JSDoc eg. `/**` or just copy/paste the docs directly.

Reply:
I was just copying what I've seen throughout the typings... in fact none of the links in that file use JSDoc except the one that I added :/

Reply:
Yeea, docs are a little bit inconsistent but every bit helps!